pkg/task/meta.go

```diff
@@ -154,7 +166,7 @@ func (m *metaTaskExecutor) getRunNamespace() string {
 func (m *metaTaskExecutor) getRetry() (attempts int, interval time.Duration) {
 	retry := m.metaTask.Retry
 	// "attempts,interval" format
-	defRetry := "1,0s"
+	defRetry := "0,0s"
```

Comment:
Why this change?

Reply:
retry & original execution are treated as separate. retry will happen only if original execution has `verify error`. retry will happen if retry value is set.